### ISSUE TITLE
Can you package it to jar or aar, then push it to the moven central or jcenter?

### ISSUE THREAD

**Issue by snowdream** [andrewhayden/archive-patcher#9]
@andrewhayden 

Can you package it to jar or aar, then push it to the moven central or jcenter?

**Comment by juliantoledo** [comment]
Andrew left Google a couple of month ago.

That is a great idea, I'm OOO but I can take a look next week.

**Comment by snowdream** [comment]
@juliantoledo thank you.

**Comment by andrewhayden** [comment]
@juliantoledo ping, this sounds like a good thing to do. Any chance you have time to work on it?

**Comment by juliantoledo** [comment]
@andrewhayden I haven't been able to put time into it. It is a great idea and we should do it soon.

**Comment by theimpostor** [comment]
In the meantime, there is: https://jitpack.io/#andrewhayden/archive-patcher/1.0

**Comment by andrewhayden** [comment]
@juliantoledo ping again, any progress?

**Comment by andrewhayden** [comment]
@juliantoledo Any update?

**Comment by juliantoledo** [comment]
Yes, I just added a few minutes ago another commit to the pull request for the sourcesJar generation. I still need to test that it works on maven central before we can merge it.

**Comment by andrewhayden** [comment]
@juliantoledo ping

**Comment by juliantoledo** [comment]
I made it public here:
https://bintray.com/juliantoledo/Maven/archive-patcher
https://bintray.com/bintray/jcenter?filterByPkgName=archive-patcher

**Comment by fzyzcjy** [comment]
@juliantoledo Hi it seems that the package already go to 1.1, thus could you please update the file in bintray? Thanks!
